### PR TITLE
[SDK-75] Add account linking listener events

### PR DESCRIPTION
**Pull request by HarrisonHough**
## Description

-   Adds listeners for the account linking events
- added event callbacks
- minor refactoring

#### Added

-   support for listening to account linking events

#### Updated

-   refactor of JS event handling

## How to Test

-   Add steps to locally test these changes

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.
-   [ ] QA Testing is updated.

## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.

[SDK-75]: https://ready-player-me.atlassian.net/browse/SDK-75?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Runtime/WebViewEventNames.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+public static class WebViewEventNames
```
Comment (srcnalt) [inline Runtime/WebViewEventNames.cs:2]: namespace missing

File: Runtime/WebViewPanel.cs [hunk 2]
```diff
@@ -121,30 +119,42 @@ private void OnWebMessageReceived(string message)
...
             {
                 SDKLogger.AvatarLoaderLogger.Log(TAG, $"--- Message is not JSON: {message}\nError Message: {e.Message}");
             }
         }
 
+        private void HandleEvents(WebMessage webMessage)
+        {
+            switch (webMessage.eventName)
+            {
+                case WebViewEventNames.AVATAR_EXPORT:
```
Comment (srcnalt) [inline Runtime/WebViewPanel.cs:134]: I think moving the event here and pre-subbing to it in start would make more sense.

```suggestion
                case WebViewEventNames.AVATAR_EXPORT:
                   OnAvatarCreated?.Invoke(url);
```

Comment (HarrisonHough) [reply Runtime/WebViewPanel.cs:134]: Hmm what abou this instead? 

```cs

            switch (webMessage.eventName)
            {
                case WebViewEventNames.AVATAR_EXPORT:
                    OnAvatarCreated?.Invoke(webMessage.GetAvatarUrl());
                    HideAndClearCache();
                    break;
....

        private void HideAndClearCache()
        {
            if (urlConfig.clearCache)
            {
                loaded = false;
                webViewObject.Reload();
            }

            SetVisible(false);
        }
```

Comment (HarrisonHough) [reply Runtime/WebViewPanel.cs:134]: or maybe this 

```cs
                case WebViewEventNames.AVATAR_EXPORT:
                    OnAvatarCreated?.Invoke(webMessage.GetAvatarUrl());
                    HandleClearCache();
                    SetVisible(false);

```

Comment (srcnalt) [reply Runtime/WebViewPanel.cs:134]: Could be, I missed that this event requires func with string param.

**Pull request merged**